README.md

```diff
@@ -38,36 +38,47 @@ Once the dependencies are installed, run
 
 ## Executables
 
-Go Ethereum comes with several wrappers/executables found in 
-[the `cmd` directory](https://github.com/ethereum/go-ethereum/tree/develop/cmd):
+Go Ethereum comes with several wrappers/executables found in the `cmd` directory.
```

Comment:
Sure, first thing in the morning.
On May 2, 2016 18:34, "Felix Lange" <notifications@github.com> wrote:

> In README.md
> <https://github.com/ethereum/go-ethereum/pull/2505#discussion_r61755295>:
>
> > @@ -38,36 +38,47 @@ Once the dependencies are installed, run
> >
> >  ## Executables
> >
> > -Go Ethereum comes with several wrappers/executables found in
> > -[the `cmd` directory](https://github.com/ethereum/go-ethereum/tree/develop/cmd):
> > +Go Ethereum comes with several wrappers/executables found in the `cmd` directory.
>
> Cool, so will you please change it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/go-ethereum/pull/2505/files/53b762fb53099be6c99249717b3958bda30e6bb8#r61755295>
>